internal/watcher/watcher.go

```diff
@@ -63,6 +52,7 @@ func GetAllInDirectory(dir string, ignoredExtensions []string) (
 	return files, directories, nil
 }
 
+// WatchFuncProvider for interfacing WatchFn for WatchDirectories
```

Comment:
What does this sentence mean?

Reply:
Fixed by https://github.com/mattermost/mattermost-plugin-nagios/pull/31/commits/e9d81e1dc1d8a7df42365816b2c1900c497136bd

Reply:
> Thanks for the update 👍
> 
> Two blocker from my side:
> 
> 1. As mentioned in [#31 (comment)](https://github.com/mattermost/mattermost-plugin-nagios/pull/31#issuecomment-917878613) the code should reject files how are larger the 100KB.
> 2. Could you please add a test 1. and the ignore case?

Added ignore test: https://github.com/mattermost/mattermost-plugin-nagios/pull/31/commits/ec950370c5676edb753bb14d7d985d1cc6b18c02